src/elements/dropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { Menu } from '@headlessui/react';
+import { ChevronDownIcon } from '@heroicons/react/solid';
+import React, { FC, useEffect } from 'react';
+
+function classNames(...classes: string[]) {
```

Comment:
Since you're moving this function all over the place I'd extract it to its own file and import it anywhere you use it.